mmv1/third_party/terraform/services/apphub/data_source_apphub_discovered_workloads.go

```diff
@@ -0,0 +1,177 @@
+package apphub
+
```

Comment:
Can you please add the documentation for this data source in the folder https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/website/docs/d?

Reply:
Done.